### PR TITLE
Flux Zenoss pkg

### LINKED ISSUE #2278: Flux: Zenoss package
**PR:** https://github.com/influxdata/flux/pull/3524

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2278 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by sanderson** [comment]
@alespour You're welcome to take a look at this PR.

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/endpoint.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  type: "rpc",
+  tid: 1
+)
+```
+
+## Parameters
+
+### url
+({{< req >}})
+Zenoss events [public endpoint](https://help.zenoss.com/zsd/RM/configuring-resource-manager/enabling-access-to-browser-interfaces/creating-and-changing-public-endpoints) URL.
```
Comment (alespour) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/endpoint.md:34]: "Zenoss router endpoint URL" is more accurate `url` parameter description I would say  now after I've read https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/anatomy-of-an-api-request#AnatomyofanAPIrequest-RouterURL (I wish I had found it before, sorry!).

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/event.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  collector: "",
+  message: ""
+)
+```
+
+## Parameters
+
+### url
+({{< req >}})
+Zenoss events [public endpoint](https://help.zenoss.com/zsd/RM/configuring-resource-manager/enabling-access-to-browser-interfaces/creating-and-changing-public-endpoints) URL.
```
Comment (alespour) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/event.md:40]: Like in `endpoint()` function, "Zenoss router endpoint URL" seems more correct to me.

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/endpoint.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+_**Data type:** String_
+
+### action
+Zenoss [router name](https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/anatomy-of-an-api-request#AnatomyofanAPIrequest-RouterURL).
+Default is `"EventsRouter"`.
+
+_**Data type:** String_
+
+### method
+[EvensRouter method](https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/codebase/routers/router-reference/eventsrouter).
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/endpoint.md:59]:
```suggestion
[EventsRouter method](https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/codebase/routers/router-reference/eventsrouter).
```

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/event.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+_**Data type:** String_
+
+### action
+Zenoss [router name](https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/anatomy-of-an-api-request#AnatomyofanAPIrequest-RouterURL).
+Default is `"EventsRouter"`.
+
+_**Data type:** String_
+
+### method
+[EvensRouter method](https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/codebase/routers/router-reference/eventsrouter).
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/event.md:65]:
```suggestion
[EventsRouter method](https://help.zenoss.com/dev/collection-zone-and-resource-manager-apis/codebase/routers/router-reference/eventsrouter).
```

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/event.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+    url: "https://tenant.zenoss.io:8080/zport/dmd/evconsole_router",
+    username: username,
+    username: password,
+    device: lastReported.host,
+    component: "CPU",
+    eventClass: "/App",
+    severity:
+      if lastReported._value < 1.0 then "Critical"
+      else if lastReported._value < 5.0 then "Warning"
+      else "Info",
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/event.md:165]: Should we add clear to the example?

**Review by kelseiv — APPROVED** [review]

**Pull request merged**